packages/esm-patient-allergies-app/src/dashboard.meta.ts

```diff
@@ -1,5 +1,5 @@
 export const dashboardMeta = {
   slot: 'patient-chart-allergies-dashboard-slot',
   columns: 1,
-  path: 'Allergies',
+  title: 'Allergies',
```

Comment:
So, I get that this will work, but I actually had a separate motivation in adding a `path` property, specifically, I actually think it would be more consistent to allow the URL to vary regardless of the title. `Vitals & Biometrics` is great display text, but gets a bit weird as soon as you start realising how much URL encoding and decoding we need to do and is not obviously more useful in that format than, e.g., `vitals-and-biometrics` as a URL slug.